x/auction/keeper/keeper.go

```diff
@@ -9,31 +9,51 @@ import (
 	"github.com/umee-network/umee/v6/x/ugov"
 )
 
+type SubAccounts struct {
+	Rewards    []byte
+	RewardsBid []byte
+}
+
```

Comment:
can we move this `SubAccounts` into types

Reply:
We don't have `types` package, and we don't want to have it. It's part of what the keeper expects, so I prefer to have it here